Cargo.lock

```diff
@@ -244,13 +244,13 @@ dependencies = [
 
 [[package]]
 name = "actix_derive"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
I doesn't look like this is a required upgrade for this change, lets keep crate upgrades in self-contained PRs so that they can be reviewed on their own merit.

Reply:
It seems that it was pulled by a new clone and build... Would restore it